lib/bolt/cli.rb

```diff
@@ -752,12 +752,14 @@ def install_puppetfile(config, puppetfile, modulepath)
     end
 
     def pal
+      proj_path = config.project_name ? config.boltdir.path.to_s : nil
```

Comment:
I think it might be better to have this logic in `Bolt::Config`, where you can call `config.project_path` and it returns either `@boltdir.path.to_s` if `project_name` is set or `nil`.